### ISSUE TITLE
[Mobile] Sending PM, posts or comments when no network should fail

### ISSUE THREAD

**Issue by AugierLe42e** [diaspora/diaspora#5566]
Currently, when sending a long message with unstable network, the message stays blocked infinitly. There is no way to copy it or edit it again and it is never transmitted. This requires to reload the page and write the comment again.

On mobile version, there should be an uptime for each transmission. If the message cannot be send until then, the UI should display a warning and come back to the edition of the message.

**Comment by goobertron** [comment]
Yes, this is annoying behaviour. Related to #3688, although for a different part of the app.

**Comment by svbergerem** [comment]
This has been fixed for conversations by #7131 and will be further improved by #7227. This still needs to be fixed for posts and comments.
